src/scoring.jl

```diff
@@ -286,7 +288,7 @@ function ck_workers_worktime(workers, shifts, workers_info)::ScoringResult
         hours_per_week::Float32 = workers_info["time"][workers[worker_no]] * WORKTIME_BASE
 
         worktime = num_weeks * hours_per_week
-        exempted_worktime = hours_per_week / NUM_WORKING_DAYS * exempted_days_no
+        exempted_worktime = (hours_per_week / NUM_WORKING_DAYS) * (exempted_days_no + holidays)
         req_worktime = worktime - exempted_worktime
 
         act_worktime = sum(map(s -> SHIFTS_TIME[s], shifts[worker_no, :]))
```

Comment:
Jakbyś miał pomysł jak to ładniej zapisać, w sensie jakiś mini refactor nazw zmiennych, to wal, bo teraz patrze na ten moj kod i mi się nie podoba xD

Reply:
Hmmm, a może jakby przejść z bazowania na tygodniach jako jednostce czasu na dni? Mogę dać w commicie z poprawkami jakbym to widział

Reply:
Chociaż w sumie nie wiem czy by to nie wprowadziło więcej zamętu niż zysku w innych miejscach xD

Reply:
nie wiem, czy to dobry pomysł, nadgodziny oraz niektóre ograniczenia rozpatrujemy tygodniowo